pkg/ipxe/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM lfedge/eve-alpine:6.2.0 as build
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:6.2.0
```

Comment:
If you want to enforce that it _always_ passes the arg (you might not want to, but if you so), just set the ARG to be blank:

```Dockerfile
ARG EVE_BUILDERIMAGE
FROM ${EVE_BUILDER_IMAGE} as build
```

Reply:
Just thinking out loud. Perhaps it is worthy to enforce it, actually. Could potentially reduce the space for a mistake. If for whatever obscure reason the argument is not passed to one of the docker files, it will be build with a different image.

Reply:
I actually do NOT want to enforce that. The default is completely fine. All this change does is make it tweakable from the command line -- that's all. Very, very simple refactoring that doesn't intend to change any semantics.

Reply:
I am of two minds here. If we expect people to build this by just running `docker build` or similar, then having a default is fine. If we _always_ expect people to build it using some wrapping script, or _always_ expect them to pass in the image explicitly, then enforcing it would be good.